README.md

```diff
@@ -1,61 +1,2580 @@
+  && cat /mnt/tmp/newfile.txt'
+Temp file!
+```
+This time it worked correctly, because we eased the restriction on the container
+to allow the `CAP_SYS_ADMIN` capability. Note that when passing the capability,
+the `CAP_` prefix is dropped.
+
+It's important to note that should your container require elevated permissions
+or capabilities above that of the default controls, you should find the minimal
+number of changes required for the functionality required. This way your
+container will continue to be secured as much as possible from malicious code.
+
+### 11. Docker Registries and Repositories
+
+Once you've built a working docker image that contains all the functionality you
+need, it will be stored in the local Docker registry on the machine that you've
+carried out development on.
+
+We've already seen the local registry, it's the store for images, the list of
+which can be seen by running the following command:
+```
+$ docker images
+REPOSITORY                        TAG                 IMAGE ID            CREATED             SIZE
+simple-docker-app                 latest              b8a722866d8c        3 days ago          273MB
+simple-docker-app                 v0.0.2              5e2c8c1cb9d9        5 days ago          273MB
+simple-docker-app                 v0.0.1              8a65ba22f11f        6 days ago          273MB
+balenalib/intel-nuc-debian-node   latest              ca37795ac1cc        2 weeks ago         270MB
+```
+This local repository stores all of the images that have been built or used
+when building the images. But how do we now share these images, so that others
+can use them? Well, think back to our Dockerfile, specifically the first line
+where we determined which base image to use:
+```
+FROM balenalib/intel-nuc-debian-node:latest
+```
+As we know, this tells Docker which image to base the rest of the image we're
+building on. As you may also remember, on the first build, this downloaded the
+image. This download came from a remote Docker registry, which Docker itself
+hosts, called Docker Hub. Any user with an account on Docker Hub can upload
+images they've built so that other users can download them and run them, or use
+them as base images. Docker has both public and private registries, depending
+on the type of account (unpaid accounts only allow the maximum of one private
+repository, with all others being public, whereas paid accounts allow more
+private repositories). Docker Hub is the default Docker registry that is used
+when installing Docker, but other registries can also be used including private
+ones that you can run yourself. We'll discuss this in a future Masterclass.
+
+We discussed tags in a previous section, where we noted that tags that we give
+images are of the format `repository:tag`. A repository name denotes a specific
+location in a registry for a group of images. On Docker Hub, repositories are
+usually denoted by the name of a user or organisation using the registry and
```

Comment:
Maybe for ease of reading forward slash, though as the example follows feel free to ignore that.

Reply:
Changed from solidus.